src/wallet/rpcdump.cpp

```diff
@@ -693,8 +693,12 @@ UniValue dumpwallet(const JSONRPCRequest& request)
 
     EnsureWalletIsUnlocked(pwallet);
 
+    boost::filesystem::path filePath = request.params[0].get_str();
+    filePath = boost::filesystem::absolute(filePath);
```

Comment:
I suggest using `canonical` instead of `absolute` if you really want to show users where their file ends up. E.g. `/home/user/mydir/../somedir/wallet.dump` isn't something user can quickly parse